fdbserver/storageserver.actor.cpp

```diff
@@ -3320,6 +3320,7 @@ void changeServerKeys(StorageServer* data,
 		} else if (!dataAvailable) {
 			// SOMEDAY: Avoid restarting adding/transferred shards
 			if (version == 0) { // bypass fetchkeys; shard is known empty at version 0
+				TraceEvent("ChangeServerKeysEmptyRange", data->thisServerID);
```

Comment:
can we output range as well? that helps to identify which lost/empty ranges are added/assigned to which SSes.

Reply:
Done.